samples/http-cloudevents/server.py

```diff
@@ -23,6 +23,18 @@
 def home():
     # create a CloudEvent
     event = from_http(request.get_data(), request.headers)
+    print(event)
```

Comment:
What does the output look like?

If we print the whole event, do we really need to print `event['source']`, `event['id']` etc?

Reply:
I was mostly trying to showcase access to event attributes. If anything, I'd remove the print(event). Open to thoughts on this!